.github/workflows/contracts.yaml

```diff
@@ -0,0 +1,76 @@
+name: Solidity
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  contracts-build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+          cache: "yarn"
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build contracts
+        run: yarn build
+
+      - name: Run tests
+        run: yarn test
+
+  contracts-lint:
```

Comment:
Maybe we don't need this job as `yarn lint` is a part of `yarn format` we check in the other workflow?

Reply:
Yeah, good call. So maybe we'll drop this job and rename `code-format` job to `code-lint-and-format`? Linting is about static code analysis to flag programming errors, bugs, stylistic errors, and suspicious constructs. Formatting is just about the convention: new lines, whitespaces, semicolons, and so forth.

Reply:
💯